content/docs/sidebar.json

```diff
@@ -111,6 +111,21 @@
             "slug": "docker",
             "label": "Docker",
             "source": "docker.md"
+          },
+          {
+            "slug": "requirements",
+            "label": "Requirements",
+            "source": "requirements.md"
+          },
+          {
+            "slug": "venv",
+            "label": "Virtual Environments",
```

Comment:
Is there a logical sorting of this menu? pip/whl , Docker, pip -r, venv, conda... No clear logic that I can see. Let's make it alphabetical at least?

Reply:
done, the new order is conda, docker, pip, requirements, venv

Reply:
Thanks